### ISSUE TITLE
Question about the code for loss

### ISSUE THREAD

**Issue by yipliu** [varsha33/LCL_loss#11]
Hi,

I didn't understand much about [this line in code](https://github.com/varsha33/LCL_loss/blob/9b4397cec0e0270ecf10c891b8da23ac5c061043/loss.py#L141). Is this an engineering need?

```python
        logits_max, _ = torch.max(anchor_dot_contrast, dim=1, keepdim=True)
        logits = anchor_dot_contrast - logits_max.detach()
```
